.golangci.yml

```diff
@@ -62,6 +64,12 @@ linters-settings:
     enable-all: true
   ginkgolinter:
     forbid-focus-container: true
+  importas:
+    alias:
+    - pkg: github.com/projectcontour/contour/apis/projectcontour/(v\w+)
+      alias: contour_$1
```

Comment:
considered `projectcontour` here too, but this is shorter and we already use variants of this in many places

Reply:
SGTM